benchmarks/annulus/annulus.cc

```diff
@@ -1,8 +1,10 @@
+#include <aspect/simulator.h>
```

Comment:
Is it necessary to have the changes to annulus.cc in this PR? Could you separate it into its own PR?